### PR TITLE
Add channel group and option to choose version for policy tags in upgrade roles cmd

### PR DESCRIPTION
**Pull request by gdbranco**
# What
User couldn't upgrade roles to tag policies with the nightly release

# Why
Allow option to choose nightly channel group when upgrading policy tags

### CODE REVIEW

File: cmd/upgrade/roles/cmd.go [hunk 1]
```diff
@@ -473,6 +496,9 @@ func upgradeAccountRolePoliciesFromCluster(
 			return err
 		}
 		promptString := fmt.Sprintf("Upgrade the '%s' role policy latest version (%s) ?", roleName, policyVersion)
+		if isVersionChosen {
```
[the remark this answers is not in the document]
Comment (gdbranco) [reply cmd/upgrade/roles/cmd.go:499]: I actually prefer the other way, do we have a standard for this?

[the remark this answers is not in the document]
Comment (oriAdler) [reply cmd/upgrade/roles/cmd.go:499]: Not that I'm aware of, so let's keep it as it is.

File: cmd/upgrade/roles/cmd.go [hunk 2]
```diff
@@ -106,7 +124,10 @@ func run(cmd *cobra.Command, argv []string) error {
 		os.Exit(1)
 	}
 
-	policyVersion, err := ocmClient.GetDefaultVersion()
+	policyVersion := args.policyUpgradeversion
```
[the remark this answers is not in the document]
Comment (gdbranco) [reply cmd/upgrade/roles/cmd.go:127]: Does make sense, will do

**Pull request merged**